CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.1.0] - 2023-03-30
+
+### Added
+
+- Adds optional `vintage_start_year` and `vintage_end_year` field to `order` creation
+- Adds optional `vintage_start_year` and `vintage_end_year` field to `order_line_item` create and update
+- Adds optional `vintage_start_year` and `vintage_end_year` field to `inventory` creation
+- Adds `vintage_start_year` and `vintage_end_year` field to `order` response
+- Adds `vintage_start_year` and `vintage_end_year` field to `order_line_item` response
+- Adds optional `carrier_scac` field to `patch.estimates.create_road_shipping_estimate`
```

Comment:
@holtbp this should have been updated when that was released, right?

Reply:
Correct

Reply:
cc @boconnell2 to update the docs as well